### DISCUSSION TITLE
File size limits?

### DISCUSSION THREAD

**Discussion by SD777x** [NginxProxyManager/nginx-proxy-manager#2973] [Q&A]
I'm not sure where to go with this, hoping this community can help. I'm pretty much network illiterate so please speak to me like I'm in Network Kindergarten. I'm using Nginx Proxy Manager as a reverse proxy server for the BlueBubbles app. The BlueBubbles app allows you to use Apple's iMessage on an Android or Windows device. The initial issue is that BlueBubbles uses Cloudflare, but Cloudflare has a 100mb limit on uploaded files so I was unable to send videos larger than 100mb. As Nginx is using my computer as a server (via DDNS provided by my router), from what I understand their internal limit is 1gb, which should allow me to send larger videos.  I've verified with the BB devs that they have coded a 1gb limit to the BlueBubbles server for files, so the bottleneck shouldn't be on their end. In any case [here ](https://github.com/BlueBubblesApp/bluebubbles-server/blob/master/packages/server/src/server/services/httpService/index.ts#L112-L115) is the Bluebubbles server code. 

Nginx Proxy Manager is setup on MacOS (Ventura) via Docker and everything in the BlueBubbles app works well except I'm still running into a 100mb limit when sending videos. The BlueBubbles devs say the issue is on Nginx's end because their internal limit is 1gb and there must be some setting or parameter in Nginx that I have not set. Looking through the admin settings I don't see anything that could be the culprit. Hoping someone on here can figure out a setting or parameter that can make this work.

Here is BlueBubbles documentation on setting up the reverse proxy: https://docs.bluebubbles.app/server/advanced/byo-proxy-service-guides/nginx-proxy-manager

So my question is, does Nginx Proxy Manager impose any upload limits on file size?  If so, is there a way to allow for increased file sizes?

For reference my Docker compose file:
version: '3.8' 
services: 
  app: 
    image: 'jc21/nginx-proxy-manager:latest' 
    restart: unless-stopped 
    ports: 
      # These ports are in format <host-port>:<container-port> 
      - '80:80' # Public HTTP Port 
      - '443:443' # Public HTTPS Port 
      - '81:81' 
      - # Admin Web Port 
      - # Add any other Stream port you want to expose 
      - # - '21:21' # FTP

    # Uncomment the next line if you uncomment anything in the section
    # environment:
      # Uncomment this if you want to change the location of
      # the SQLite DB file within the container
      # DB_SQLITE_FILE: "/data/database.sqlite"
    
      # Uncomment this if IPv6 is not enabled on your host
      # DISABLE_IPV6: 'true'
    
    volumes:
      - ./data:/data
      - ./letsencrypt:/etc/letsencrypt

**Comment by BigBen2003** [comment]
In PHP wird die maximale Dateigröße mit upload_max_filesize festgelegt. In Docker gibt es auch eine PHP.INI Datei, in der die Werte verändert werden können.

Vielleicht hilft das etwas weiter.
